components/finder.rst

```diff
@@ -82,7 +82,7 @@ directory to use for the search::
 Search in several locations by chaining calls to
```

Comment:
this no longer holds true.

I propose to merge both code examples (this one and the one on line 80):

```rst
The location is the only mandatory criteria. It tells the finder which
directory to use for the search::

    $finder->in(__DIR__);

    // search in several locations
    $finder->in(array(__DIR__, '/elsewhere'));
    $finder->in(__DIR__)->in('/elsewhere');

    // use wildcard characters (it has to resolve to at least one directory path)
    $finder->in('src/Symfony/*/*/Resources');

Exclude directories from matching with the [...]
```